x/gov/keeper/grpc_query.go

```diff
@@ -54,6 +55,57 @@ func (q queryServer) Proposal(ctx context.Context, req *v1.QueryProposalRequest)
 	return &v1.QueryProposalResponse{Proposal: &proposal}, nil
 }
 
+// ProposalVoteOptions returns the proposal votes options
+// It returns the stringified vote options if the proposal is a multiple choice proposal
+// Otherwise it returns the generic vote options
+func (q queryServer) ProposalVoteOptions(ctx context.Context, req *v1.QueryProposalVoteOptionsRequest) (*v1.QueryProposalVoteOptionsResponse, error) {
+	if req == nil {
+		return nil, status.Error(codes.InvalidArgument, "invalid request")
+	}
+
+	if req.ProposalId == 0 {
+		return nil, status.Error(codes.InvalidArgument, "proposal id can not be 0")
+	}
+
+	ok, err := q.k.Proposals.Has(ctx, req.ProposalId)
+	if err != nil {
+		return nil, status.Error(codes.Internal, err.Error())
+	}
+
+	if !ok {
+		return nil, status.Errorf(codes.NotFound, "proposal %d doesn't exist", req.ProposalId)
+	}
+
+	optionSpam := "SPAM"
```

Comment:
should we lift this outside the function and make it a const? no need to set the value on each call